eth/beacon/state_machines/base.py

```diff
@@ -74,7 +74,7 @@ def logger(self):
     @classmethod
     def get_block_class(cls) -> Type[BaseBeaconBlock]:
         """
-        Return the :class:`~eth.beacon.types.blocks.BeaconBlock` class that this
+        Returns the :class:`~eth.beacon.types.blocks.BeaconBlock` class that this
```

Comment:
I believe the original `Return` was the proper format we are using for docstrings (cc @cburgdorf)

Reply:
I don't have any preferences! 😆 Searched for `eth` with this PR, there are 81 `Returns` and 64 `Return` now. I can change it back.

Reply:
Correct, we try to streamline docs to always be imperative present tense. I *believe* that is what most documentation uses but it is at least inline with what the official Python docs use.

Here's a rule of thumb. If this sounds correct, it is correct:

*If this method is called it will: `<doc-string>`*

Example:

If this method is called it will *return the `BeaconBlock` class*

Reply:
Just noting that this is the same for git commit messages which at least makes it easier for me to remember.